examples/jss/src/index.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { render } from 'react-dom';
+import jss from 'jss';
+import preset from 'jss-preset-default';
+
+jss.setup(preset());
+
+const globalStyles = {
+  // Global styles
+  html: {
+    color: '#292f33',
+    fontFamily: '"Helvetica Neue", Helvetica, Arial, sans-serif',
+    fontSize: '14px',
+    lineHeight: '1.3125',
+  },
+  a: {
+    textDecoration: 'none',
+    color: '#1da1f2',
+  },
+  svg: {
+    fill: 'currentColor',
+    height: '1.25em',
+  },
+  '@media screen and (min-width: 360px)': {
+    html: {
+      fontSize: '15px',
+    },
+  },
+  '@media screen and (min-width: 600px)': {
+    html: {
+      fontSize: '16px',
```

Comment:
jss-default-unit allows us to have numeric values here as well, px is added automatically.